andela/hc-wits-kla
# [Feature 158175328] Add Directions for checks creation and management

#### What does this PR do?
Make a user interface rework on the landing page to inform the users that they can create and manage checks as a team (i.e. allows for team access)

#### Description of Task to be completed?
*  Add Three Screenshots and descriptions of how a user can access for team access

#### How should this be manually tested?
* Clone the branch, ```git clone https://github.com/andela/hc-wits-kla/tree/ft-onboarding-content-158175321```
* In your terminal, ```cd hc-wits-kla```
* In the project directory, activate the environment, ```source hc-venv/bin/activate```
* Install requirements ```pip install requirements.txt```
* Make migrations ```python manage.py makemigrations```
* Run migrations ```python manage.py migrate```
* Serve the application ```python manage.py runserver```
* In the browser, visit, 127.0.0.1:8000/
* On the welcome landing page, check out the last three (3) screenshots that provide for team access insights

#### What are the relevant pivotal tracker stories?
[#158175328](https://www.pivotaltracker.com/story/show/158175328)

#### Screenshots (if appropriate)

TEAM ACCESS INSIGHTS
[Can be found at](http://127.0.0.1:8000/) 

<img width="1341" alt="screen shot 2018-07-18 at 10 02 42" src="https://user-images.githubusercontent.com/39955231/42865312-f26f770e-8a71-11e8-8607-db3facdc5b2a.png">

<img width="1285" alt="screen shot 2018-07-18 at 10 02 55" src="https://user-images.githubusercontent.com/39955231/42865314-f29dbf56-8a71-11e8-9316-5f930ee3acf0.png">
